R/approach_vaeac.R

```diff
@@ -781,7 +781,7 @@ vaeac_train_model_auxiliary <- function(vaeac_model,
       train_vlb = as.array(train_vlb$cpu()),
       val_iwae = as.array(val_iwae$cpu()),
       val_iwae_running = as.array(val_iwae_running$cpu()),
-      parameters = last_state
```

Comment:
Would prefer it to be the way it was before, but rather change `last_epoch` to `last_state` in the one line in the roxygen of `vaeac_print_train_summary`. Then no other changes are needed